app/presenters/document/field_group_presenter.rb

```diff
@@ -169,5 +163,25 @@ def section_nested_hash(mappings, subject = document)
         end
       end
     end
+
+    private
+
+    def entity_section?(section)
+      section[:entity_name] == 'agents' && section[:entity_proxy_field] == 'dcCreator'
```

Comment:
1. Why has this been limited to dcCreator?
2. The method name is misleading. Other sections are entity sections, but will not yet be linked to from the field value displayed on the item page because we only have entity pages for agents thus far. Consider renaming the method something like `linkable_entity_section?`